recent travis builds fail with pymc3 import error

e.g., https://travis-ci.org/blei-lab/edward/jobs/193551347
```==================================== ERRORS ====================================
________ ERROR collecting tests/test-models/test_pymc3model_log_prob.py ________
ImportError while importing test module '/home/travis/build/blei-lab/edward/tests/test-models/test_pymc3model_log_prob.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test-models/test_pymc3model_log_prob.py:7: in <module>
    import pymc3 as pm
E   ImportError: No module named pymc3
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 6.19 seconds ============================
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -29,7 +29,7 @@ install:
   - pip install pep8 pytest-pep8 pytest-xdist
   - pip install python-coveralls pytest-cov
   - pip install numpydoc
-  - pip install git+https://github.com/pymc-devs/pymc3
+  - pip install pymc3
   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
       pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc0-cp27-none-linux_x86_64.whl;
     elif [[ "$TRAVIS_PYTHON_VERSION" == "3.4" ]]; then

